en/02_Developer_Guides/08_Performance/index.md

```diff
@@ -5,14 +5,4 @@ introduction: Make your applications faster by learning how to write more scalab
 icon: tachometer-alt
 ---
 
-# Performance
-
-The following guide describes the common ways to speed your Silverstripe CMS website up. The general rules for getting
```

Comment:
The opcache stuff has been moved to both 00_Server_Requirements and 08_Other_Optimisations